### PR TITLE
Update quarkiverse guide links

### PR DESCRIPTION
**Pull request by marko-bekhta**
I also noticed that Dynamo and S3 guides were duplicated in data and cloud categories so I've combined those... 
Let me know if this makes sense

### CODE REVIEW

**Comment by yrodiere** [comment]
Makes sense to me, thanks. I'll let @gsmet merge though, as he generally notices a problem I completely overlooked just after I merge things like this ;D

**Review by gsmet — APPROVED** [review]

**Comment by gsmet** [comment]
Yeah, the links for 2.13 are probably not ideal but we don't have any history on the Quarkiverse so we will have to live with it.

Thanks!

**Comment by marko-bekhta** [comment]
😭 😔 
image

Now AWS guides are back to using `dev` instead of `main`

**Pull request merged**
